test/controllers/admin_controller_test.rb

```diff
@@ -29,4 +29,30 @@ class AdminControllerTest < ActionController::TestCase
 
     assert_redirected_to new_session_path
   end
+
+  test "GET new renders" do
+    sign_in_as(users(:admin))
+
+    get :new
+
+    assert_response :ok
+  end
+
+  test "POST create makes an approved admin" do
+    sign_in_as(users(:admin))
+
+    password = 'blabla'
+
+    assert_difference 'Admin.count', +1 do
+      post(:create, admin: {
+        email: 'admin@test.com',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.